recipes/erlang/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+export LIBRARY_PATH="${PREFIX}/lib:${LIBRARY_PATH}"
+export LD_LIBRARY_PATH="${PREFIX}/lib:${LD_LIBRARY_PATH}"
+export ERL_TOP="$(pwd)"
+./configure --with-ssl="${PREFIX}" --prefix="${PREFIX}" --without-javac
```

Comment:
You were completely right to ask. Now that you have added these things let's see how they go. If they cause problems, I'm happy to see either or both of these changes removed and still merge.